Makefile

```diff
@@ -0,0 +1,13 @@
+BASE_NAME     = meta-azure-service-broker
+VERSION      ?= $(shell git describe --always --abbrev=7 --dirty)
+MUTABLE_TAG  ?= canary
+IMAGE         = $(REGISTRY)$(BASE_NAME):$(VERSION)
+MUTABLE_IMAGE = $(REGISTRY)$(BASE_NAME):$(MUTABLE_TAG)
+
+docker-build:
```

Comment:
You can make those PHONY targets. https://www.gnu.org/software/make/manual/html_node/Phony-Targets.html